Sources/Metaplex/Programs/TokenMetadata/Accounts/MetadataAccount.swift

```diff
@@ -135,27 +151,38 @@ public struct MetaplexData: BorshCodable, BufferLayout {
     public static var BUFFER_LENGTH: UInt64 = 10
 
     public init(name: String, symbol: String, uri: String, sellerFeeBasisPoints: UInt16, hasCreators: Bool, addressCount: UInt32, creators: [MetaplexCreator]) {
```

Comment:
When we initialize from this `init` are these values already trimmed? If so there's a chance serializer will break since it expects non-trimmed values. 

Should we write a specific sized block (do we know the size?) in the serializer instead of having the private "true" data and public trimmed data?

Reply:
Perfect question. This is address on the new serializer. which has fixable and non fixable types. This is good for know. Let see how the program interacts since there is a change its handle in their end